### PR TITLE
data jobs

### PR DESCRIPTION
**Pull request by behzadh**
Changing the destination of data analysis jobs to ANY again

### CODE REVIEW

File: solid/templates/runscript.sh [hunk 1]
```diff
@@ -87,7 +87,7 @@ echo -e "\n"
 
 rm ${inputfile}
 rm mysql-connector-c++-1.1.8-linux-glibc2.5-x86-64bit.tar.gz
-rm S2-histos_cycleMode_${jobnumber}_$runNUMBER.root S2-tuple_${jobnumber}_$runNUMBER.root log_${jobnumber}_$runNUMBER.txt
```
Comment (alexanderrichards) [inline solid/templates/runscript.sh]: Do you not need the jobnumber and runnumber any more?

File: solid/ParametricJobs.py [hunk 2]
```diff
@@ -55,6 +55,7 @@ def _setup_dirac_job(self, job, tmp_runscript):
             if self.solidsim_macro == "cosmicsSim.mac":
                 
                 inputdata_lfns = ['/solidexperiment.org/MC/cosmicGeneratorsFiles/2017/atm-n-1000files/Gordon-Events-%d.txt' % i for i in range(1000)]
+                print(inputdata_lfns)
```
Comment (alexanderrichards) [inline solid/ParametricJobs.py]:
```suggestion
```

**Review by alexanderrichards — CHANGES_REQUESTED** [review]

**Review by alexanderrichards — COMMENTED** [review]
I have no way of knowing, but are you sure you don't need the `${jobnumber}_$runNumber` any more?

**Review by alexanderrichards — DISMISSED** [review]

File: solid/templates/runscript.sh (hunk 1 above)
Comment (behzadh) [reply solid/templates/runscript.sh]: yes, we need job and runnumber for saving them on the SE, for removing them we don't need them.

**Pull request merged**